hplot/hstack.go

```diff
@@ -119,10 +119,11 @@ func (hstack *HStack) DataRange() (xmin, xmax, ymin, ymax float64) {
 	for _, h := range hstack.hs {
 		for i, bin := range h.Hist.Binning.Bins {
 			sumw := bin.SumW()
+			errw := bin.ErrW()
 			xmax = math.Max(bin.XMax(), xmax)
 			xmin = math.Min(bin.XMin(), xmin)
-			ymax = math.Max(yoffs[i]+sumw, ymax)
-			ymin = math.Min(yoffs[i]+sumw, ymin)
+			ymax = math.Max(yoffs[i]+sumw+0.5*errw, ymax)
+			ymin = math.Min(yoffs[i]+sumw-0.5*errw, ymin)
```

Comment:
aha! nice catch.

but actually, I think the perhaps "right" fix is to uncomment the lines 140-148 below (the nice block prefixed with an ominous `// FIXME`) and replace the `if h.YErrs` with a loop over all histos and accumulate each of the band's DataRanges (if not nil).

what do you think?

Reply:
Indeed! It's bit more trickier than I anticipated, but I should be able to fix it in PR. I'll take case of it

Reply:
yeah, that's why I'd originally left it for later, being lazy and all :)

thanks for doing all this grunt work.